docs/api/withNavigation.md

```diff
@@ -0,0 +1,14 @@
+# withNavigation
+[`withNavigation`](/src/views/withNavigation.js) is a Higher Order Component which injects a `navigation` into wrapped Component. It's useful when you can't/don't want to pass `navigation` explicitly into nested component.
```

Comment:
perhaps

```
which passes the `navigation` prop
```

is better. also,

> It's useful when you cannot pass the `navigation` prop into the component directly, or don't want to pass it in case of a deeply nested child.